sfge/src/main/java/com/salesforce/graph/ops/expander/ApexPathCollapser.java

```diff
@@ -13,7 +12,7 @@
  * ApexDynamicPathCollapser#collapse(MethodVertex, List)}. The list of ApexDynamicPathCollapsers is
  * retrieved from the
  */
-interface ApexPathCollapser extends Indexable {
+interface ApexPathCollapser {
```

Comment:
Removed Registrable implementation portion from ApexPathCollapser. The run results look really close and seem indistinguishable from an ApexPathCollapser registry run. This has simplified a few other things.